src/NUnitFramework/framework/Internal/Execution/WorkItemQueue.cs

```diff
@@ -322,13 +322,16 @@ public void Pause()
         /// </summary>
         internal void Save()
         {
-            Pause();
+            bool isRunning = State == WorkItemQueueState.Running;
+            if (isRunning)
+                Pause();
 
             _savedState.Push(new SavedState(this));
 
             InitializeQueues();
 
-            Start();
+            if (isRunning)
+                Start();
```

Comment:
This fixes the cause of the problem. When saving the state of queues, we started all the new queues running, not just those that were already running under the control of a `WorkShift`.